Update dependency @elastic/charts to v19.1.2

markov00 opened the pull request:
## Summary

This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [@elastic/charts](https://togithub.com/elastic/elastic-charts) | dependencies | major | [`18.4.2` -> `19.1.2`](https://github.com/elastic/elastic-charts/compare/v18.4.2...v19.1.2) |


---

### Release Notes

## [19.1.2](https://github.com/elastic/elastic-charts/compare/v19.1.1...v19.1.2) (2020-05-04)


### Bug Fixes

* **partition:** elimination of zero values ([#658](https://github.com/elastic/elastic-charts/issues/658)) ([9ee67dc](https://github.com/elastic/elastic-charts/commit/9ee67dc89851e268edf79016915b973bcea6bd98)), closes [#642](https://github.com/elastic/elastic-charts/issues/642)

## [19.1.1](https://github.com/elastic/elastic-charts/compare/v19.1.0...v19.1.2) (2020-04-30)


### Bug Fixes

* render charts without series ([#657](https://github.com/elastic/elastic-charts/issues/657)) ([0c0af01](https://github.com/elastic/elastic-charts/commit/0c0af01413b00734bd3dfa13dbc3aa7571ee4240))

# [19.1.0](https://github.com/elastic/elastic-charts/compare/v19.0.0...v19.1.0) (2020-04-30)


### Features

* **partition:** treemap group text in grooves ([#652](https://github.com/elastic/elastic-charts/issues/652)) ([304dd48](https://github.com/elastic/elastic-charts/commit/304dd481b0c3195022d2a1b26887901ba56c07e0))

# [19.0.0](https://github.com/elastic/elastic-charts/compare/v18.4.2...v19.0.0) (2020-04-28)


### Bug Fixes

* tooltip container scroll issue ([#647](https://github.com/elastic/elastic-charts/issues/647)) ([f411771](https://github.com/elastic/elastic-charts/commit/f4117717690f4086805f002afb85c3a4b0d2fe22))
* **annotations:** fix alignment at the edges ([#641](https://github.com/elastic/elastic-charts/issues/641)) ([43c5a59](https://github.com/elastic/elastic-charts/commit/43c5a59e3862b6191537b73fc0ca604e79fbc992)), closes [#586](https://github.com/elastic/elastic-charts/issues/586)


### Features

* shift click legend items & partition legend hover ([#648](https://github.com/elastic/elastic-charts/issues/648)) ([ed91744](https://github.com/elastic/elastic-charts/commit/ed9174471e31df77234ea05f307b0dce79722bea))
* **brush:** add multi axis brushing ([#625](https://github.com/elastic/elastic-charts/issues/625)) ([9e49534](https://github.com/elastic/elastic-charts/commit/9e4953474db37d33f8a19dfb1ff1a5528b0f6d54)), closes [#587](https://github.com/elastic/elastic-charts/issues/587) [#620](https://github.com/elastic/elastic-charts/issues/620)


### BREAKING CHANGES

* **brush:** The type used by the `BrushEndListener` is now in the following form 
```
{
  x?: [number, number];
  y?: Array<{ groupId: GroupId; values: [number, number];}>
}
```
where `x` contains an array of `[min, max]` values, and the  `y` property is an optional array of objects, containing the `GroupId` and the values of the brush for that specific axis.

* **annotations:** In the rectangular annotation, the `y` coordinates are now inverted: the `y0` parameter of the coordinates now refers to the minimum value and the `y1` value refers to the maximum value of the y domain.

[@elastic/charts update procedure](https://github.com/elastic/elastic-charts/issues/583)

elasticmachine commented:
Pinging @elastic/datavis (Feature:ElasticCharts)

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/)
* Commit: 8208c7c9bbcb95aaf2dceddcc01a9bede257422c
* [Pipeline Steps](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/flowGraphTable) (look for red circles / failed steps)
* [Interpreting CI Failures](https://www.elastic.co/guide/en/kibana/current/interpreting-ci-failures.html)

### Failed CI Steps
* [Execute kibana-intake](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/execution/node/257/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/","number":"44043","commit":"8208c7c9bbcb95aaf2dceddcc01a9bede257422c"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44080/)
* Commit: a7d7e76c12a1f2e5b9b93c60210f8227140afe18

### History
* :broken_heart: [Build #44043](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/) failed 8208c7c9bbcb95aaf2dceddcc01a9bede257422c

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/","number":"44043","commit":"8208c7c9bbcb95aaf2dceddcc01a9bede257422c"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44080/","number":"44080","commit":"a7d7e76c12a1f2e5b9b93c60210f8227140afe18"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44383/)
* Commit: 85f0e04c7188014f47624ed425d9c72a68bd405a

### History
* :green_heart: [Build #44080](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44080/) succeeded a7d7e76c12a1f2e5b9b93c60210f8227140afe18
* :broken_heart: [Build #44043](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/) failed 8208c7c9bbcb95aaf2dceddcc01a9bede257422c

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/","number":"44043","commit":"8208c7c9bbcb95aaf2dceddcc01a9bede257422c"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44080/","number":"44080","commit":"a7d7e76c12a1f2e5b9b93c60210f8227140afe18"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44383/","number":"44383","commit":"85f0e04c7188014f47624ed425d9c72a68bd405a"}]}
PIPELINE-->

FrankHassanabad commented:
Gave this a test run within Elastic Security and am seeing this stack trace which is not present on master at the moment when you zoom in within a chart:

![error_charts_zoom](https://user-images.githubusercontent.com/1151048/80740920-0b926780-8ad6-11ea-8644-c704b2a2b8ad.gif)

Full stack trace:
```ts
Uncaught TypeError: Cannot read property 'xDomain' of undefined
    at compute_series_domains.js:14
    at resultFuncWithRecomputations (index.js:77)
    at index.js:70
    at index.js:30
    at index.js:84
    at index.js:30
    at selector (index.js:105)
    at index.js:80
    at index.js:30
    at Object.selector [as computeLegendSelector] (index.js:105)
(anonymous)	@	compute_series_domains.js:14
resultFuncWithRecomputations	@	index.js:77
(anonymous)	@	index.js:70
(anonymous)	@	index.js:30
(anonymous)	@	index.js:84
(anonymous)	@	index.js:30
selector	@	index.js:105
(anonymous)	@	index.js:80
(anonymous)	@	index.js:30
selector	@	index.js:105
push.../../node_modules/@elastic/charts/dist/chart_types/xy_chart/state/chart_state.js.XYAxisChartState.getLegendItems	@	chart_state.js:53
push.../../node_modules/@elastic/charts/dist/state/selectors/get_legend_items.js.exports.getLegendItemsSelector	@	get_legend_items.js:6
(anonymous)	@	chart_state.js:127
p	@	VM1019:1
v	@	VM1019:1
(anonymous)	@	VM1019:1
dispatch	@	redux.js:212
e	@	VM1019:1
dispatch	@	VM1019:1
(anonymous)	@	redux.js:475
(anonymous)	@	chart_container.js:86
requestAnimationFrame (async)		
ChartContainerComponent._this.handleBrushEnd	@	chart_container.js:85
```

wylieconlon commented:
Lens just merged our brushing function, I think this PR needs to be updated: https://github.com/elastic/kibana/pull/62636

markov00 commented:
@FrankHassanabad I've fixed the issue on @elastic/charts and updated the PR with the fix.

@wylieconlon I've merged master and fixed the breaking changes in Lens

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44626/)
* Commit: 7049d2e284d2c8dc37cbf9ce33843aaed0ab4aeb

### History
* :green_heart: [Build #44383](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44383/) succeeded 85f0e04c7188014f47624ed425d9c72a68bd405a
* :green_heart: [Build #44080](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44080/) succeeded a7d7e76c12a1f2e5b9b93c60210f8227140afe18
* :broken_heart: [Build #44043](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/) failed 8208c7c9bbcb95aaf2dceddcc01a9bede257422c

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/","number":"44043","commit":"8208c7c9bbcb95aaf2dceddcc01a9bede257422c"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44080/","number":"44080","commit":"a7d7e76c12a1f2e5b9b93c60210f8227140afe18"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44383/","number":"44383","commit":"85f0e04c7188014f47624ed425d9c72a68bd405a"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44626/","number":"44626","commit":"7049d2e284d2c8dc37cbf9ce33843aaed0ab4aeb"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44880/)
* Commit: d4ae3bd19469f746e58f95a0821425e769808584

### History
* :green_heart: [Build #44626](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44626/) succeeded 7049d2e284d2c8dc37cbf9ce33843aaed0ab4aeb
* :green_heart: [Build #44383](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44383/) succeeded 85f0e04c7188014f47624ed425d9c72a68bd405a
* :green_heart: [Build #44080](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44080/) succeeded a7d7e76c12a1f2e5b9b93c60210f8227140afe18
* :broken_heart: [Build #44043](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/) failed 8208c7c9bbcb95aaf2dceddcc01a9bede257422c

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/","number":"44043","commit":"8208c7c9bbcb95aaf2dceddcc01a9bede257422c"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44080/","number":"44080","commit":"a7d7e76c12a1f2e5b9b93c60210f8227140afe18"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44383/","number":"44383","commit":"85f0e04c7188014f47624ed425d9c72a68bd405a"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44626/","number":"44626","commit":"7049d2e284d2c8dc37cbf9ce33843aaed0ab4aeb"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44880/","number":"44880","commit":"d4ae3bd19469f746e58f95a0821425e769808584"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/45271/)
* Commit: 202310f410fc22cc29ed963928a32b79a5cde967

### History
* :green_heart: [Build #44880](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44880/) succeeded d4ae3bd19469f746e58f95a0821425e769808584
* :green_heart: [Build #44626](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44626/) succeeded 7049d2e284d2c8dc37cbf9ce33843aaed0ab4aeb
* :green_heart: [Build #44383](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44383/) succeeded 85f0e04c7188014f47624ed425d9c72a68bd405a
* :green_heart: [Build #44080](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44080/) succeeded a7d7e76c12a1f2e5b9b93c60210f8227140afe18
* :broken_heart: [Build #44043](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/) failed 8208c7c9bbcb95aaf2dceddcc01a9bede257422c

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44043/","number":"44043","commit":"8208c7c9bbcb95aaf2dceddcc01a9bede257422c"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44080/","number":"44080","commit":"a7d7e76c12a1f2e5b9b93c60210f8227140afe18"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44383/","number":"44383","commit":"85f0e04c7188014f47624ed425d9c72a68bd405a"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44626/","number":"44626","commit":"7049d2e284d2c8dc37cbf9ce33843aaed0ab4aeb"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/44880/","number":"44880","commit":"d4ae3bd19469f746e58f95a0821425e769808584"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/45271/","number":"45271","commit":"202310f410fc22cc29ed963928a32b79a5cde967"}]}
PIPELINE-->
